pkg/healthchecker/types/types_windows.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package types
 
 const (
-	DefaultCriCtl        = "C:/node/crictl.exe"
```

Comment:
You can configure this in the health checker config, instead of changing the default.

Reply:
Offline discussed. `C:/etc/kubernetes/node/bin/crictl.exe` is the default path in the OSS setup https://github.com/kubernetes/kubernetes/blob/1400be60ff7f01fd2799ee897a95859693c93e85/cluster/gce/windows/k8s-node-setup.psm1#L1310.

It should be fine to default to that path.